tests/test_nicenumber.py

```diff
@@ -50,3 +50,23 @@ def test_to_human():
         else:
             assert result == expected_result
 
+
+
+def test_to_color():
+    """ Test to_color() function"""
+
+    # test exception handing
+    with raises(TypeError, match=r".* int .*"):
+        nn.to_color('abc')
```

Comment:
This could be a single line, and match calls re.search, so all the extra regex stuff isn't necessary, just `'int'` will work.

```suggestion
    raises(TypeError, f, number='abc').match('int')
```